lib/registration.pm

```diff
@@ -23,6 +23,7 @@ use strict;
 use testapi;
 use utils qw(addon_decline_license assert_screen_with_soft_timeout zypper_call systemctl);
 use version_utils qw(is_sle is_caasp sle_version_at_least is_sle12_hdd_in_upgrade);
+use constant ADDONS_COUNT => 50;
```

Comment:
This can be done simpler with one `my $count` at beginning

Reply:
I guess from memory perspective it's more efficient to use constant, but I agree that we could skip part where we init variables with this value.

Reply:
I doubt, but if yes then the difference will *funny* I mean teeny tiny.

Reply:
Sure, but constant vs. variable is senseless discussion here, only thing is that we don't use constant like we should, I'm with you on that. Having `$count` instead of `ADDONS_COUNT`, so it's more if the object is immutable or not.

Reply:
IMHO is some kind of MAX_VALUE (our best estimation of number of modules than can be displayed) at least you plan to change it from outside this module, for me looks like a constant that you can assign to some variable, for example, `my $counter` inside your functions.

Reply:
Sure, but you just create global subroutine which will exist whole run, you can get this MAX_VALUE from any package. It is just disadvantage without any good reason. With this logic we can use in every test `our` $whatever_variable instead of `my`, because we just want to do so.